mqtt/mqtt-edgehub/src/command/bridge_update.rs

```diff
@@ -0,0 +1,51 @@
+use tracing::info;
+
+use mqtt3::ReceivedPublication;
+
+use mqtt_bridge::{BridgeControllerHandle, BridgeControllerUpdate};
+
+use crate::command::Command;
+
+const BRIDGE_UPDATE_TOPIC: &str = "$internal/bridge/settings";
+
+/// `BridgeUpdateCommand` is executed when `EdgeHub` sends a special packet
+/// to notify the broker that the bridge settings has changed.
+pub struct BridgeUpdateCommand {
+    controller_handle: BridgeControllerHandle,
+}
+
+impl BridgeUpdateCommand {
+    pub fn new(controller_handle: &BridgeControllerHandle) -> Self {
+        Self {
+            controller_handle: controller_handle.clone(),
+        }
+    }
+}
+
+impl Command for BridgeUpdateCommand {
+    type Error = Error;
+
+    fn topic(&self) -> &str {
+        BRIDGE_UPDATE_TOPIC
+    }
+
+    fn handle(&mut self, publication: &ReceivedPublication) -> Result<(), Self::Error> {
+        info!("received bridge update from EdgeHub.");
+        let update: BridgeControllerUpdate =
+            serde_json::from_slice(&publication.payload).map_err(Error::ParseBridgeUpdate)?;
+
+        self.controller_handle
+            .send(update)
+            .map_err(Error::SendBridgeUpdate)?;
+        Ok(())
+    }
+}
```

Comment:
What is the testing plan here? Integration tests or mocking? I think it is hard to test now since we don't expose functionality for reading these updates.

Reply:
I think mocking or both. Will follow what authz does.